flake.nix

```diff
@@ -194,6 +194,11 @@
     # kept for compat
     lib2 = d2n-lib;
 
+    flakeModule = {
```

Comment:
Should be rename this to `flakeModuleBeta`, to make it really clear, that this is subject to changes?

Reply:
That would be fine to me. I was thinking of adding a `lib.warn` but that might be too much nagging :shrug: